velox/functions/prestosql/SIMDJsonFunctions.h

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "simdjson.h"
+#include "velox/functions/Macros.h"
+#include "velox/functions/UDFOutputString.h"
+#include "velox/functions/prestosql/json/JsonExtractor.h"
+#include "velox/functions/prestosql/json/SimdJsonExtractor.h"
+#include "velox/functions/prestosql/types/JsonType.h"
+
+namespace facebook::velox::functions {
+template <typename T>
+struct SIMDJsonArrayContainsFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+  template <typename TInput>
+  FOLLY_ALWAYS_INLINE void 
+  call(bool& result, const arg_type<Json>& json, const TInput& value) {
+
+    std::string jsonData(json);
+    ParserContext ctx(jsonData.data(), jsonData.length());
+    std::string jsonpath = "";
+    result = false;
+
+    do {
+      try {
+        ctx.parseDocument();
+      } catch (simdjson::simdjson_error& e) {
+        printf(
```

Comment:
Any particular reason not to let the exception propagate?

Reply:
will fix it.